.pipelines/stages/jobs/py-publishing-job.yml

```diff
@@ -0,0 +1,161 @@
+parameters:
+- name: SpecificArtifact
```

Comment:
You should use 1ES pipeline template.
All packaging pipelines should use 1ES pipeline template.